client/app/templates/components/show-project/actions-list/ulurp-number-link.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if @action.dcpUlurpnumber}}
+  ULURP Number:
+  {{#if (in-array
+      (array "Approved" "Certified" "Referred" "Disapproved" "Terminated" "Withdrawn" "Study Complete")
+      @action.statuscode
+    )
+  }}  
+      {{#if (and (not-eq @action.statuscode this.ACTION_STATUSCODE_ACTIVE) @action.dcpSpabsoluteurl)}}
+        {{#tool-tipster
+          content='City Planning Commision Report'
+          tagName='span'
+        }}
+          <a
+            href={{concat this.host '/document' this.serverRelativeUrl}}
```

Comment:
Why not just use the `build-url` helper here? Instead of pulling it out into another component?

Reply:
I want to shift as much complex logic to the component/model/helper files as much as possible